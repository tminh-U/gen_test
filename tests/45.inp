7
4 14 12 5 4 16 5 
11
1 5
6 7
2 5
6 7
7 7
5 5
7 7
7 7
6 6
2 6
5 7
