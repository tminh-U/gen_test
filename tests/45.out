10
3
5
3
2
2
2
2
2
10
6
